README.md

```diff
@@ -94,14 +94,89 @@ The output format for Syft is configurable as well:
 syft packages <image> -o <format>
 ```
 
-Where the `format`s available are:
+Where the `formats` available are:
 - `json`: Use this to get as much information out of Syft as possible!
 - `text`: A row-oriented, human-and-machine-friendly output.
 - `cyclonedx`: A XML report conforming to the [CycloneDX 1.2 specification](https://cyclonedx.org/specification/overview/).
 - `spdx`: A tag-value formatted report conforming to the [SPDX 2.2 specification](https://spdx.github.io/spdx-spec/).
 - `spdx-json`: A JSON report conforming to the [SPDX 2.2 JSON Schema](https://github.com/spdx/spdx-spec/blob/v2.2/schemas/spdx-schema.json).
 - `table`: A columnar summary (default).
 
+## Private Registry Authentication
+
+Syft uses the [authn](https://github.com/google/go-containerregistry/tree/main/pkg/authn) package to solve for private registry access. 
+
+The config file listed in the linked authn package is where your credentials are stored when authenticating with private registries via some command like `docker login`.
+
```

Comment:
Probably worth splitting some of this up into subsections (can also be encoded into the URL to jump straight to it)
```suggestion

### Docker credentials

```

Reply:
Added `Local Docker Credentials` section